src/css/style.css

```diff
@@ -200,6 +243,13 @@ button {
 .gpii-fd-decrease .gpii-fd-icon:before {
     content: "\e604";
 }
+.gpii-fd-increase:hover,
+.gpii-fd-decrease:hover {
+    background-color: #425165; /* blue-grey, to override foundation's blue */
+    box-shadow: 0 0 0 0.04rem rgb(255,255,255), /* white */
+                0 0 0 0.6rem rgb(72, 82, 100); /*blue-grey */
```

Comment:
This looks great, @jhung, thanks for that. I couldn't figure out how to do that. Any chance you could apply this to the buttons on the other two screens, or at least a version of it? I know they're not active, but consistency would be nice.